code/xcvm/cosmwasm/tests/src/tests/suite.rs

```diff
@@ -317,29 +317,21 @@ fn load_contracts() -> XCVMContracts {
 	// to catch dependencies in the contract.  Perhaps it makes sense to always
 	// build the contracts?  Presumably if they were not changed, `cargo build`
 	// will be quick.
-	fn read_contract(env: &str, filename: &str) -> Vec<u8> {
+	fn read_contract(filename: &str) -> Vec<u8> {
 		let contracts_dir = std::path::Path::new(concat!(
 			env!("CARGO_MANIFEST_DIR"),
 			"/../../target/wasm32-unknown-unknown/cosmwasm-contracts",
 		));
-		match std::env::var_os(env) {
-			Some(path) => read(path),
-			None => read(contracts_dir.join(filename)),
-		}
+		read(contracts_dir.join(filename))
 	}
 
-	let code_interpreter = read_contract("CW_XCVM_INTERPRETER", "cw_xc_interpreter.wasm");
-	let code_gateway = read_contract("CW_XCVM_GATEWAY", "cw_xc_gateway.wasm");
-
+	let code_interpreter = read_contract("cw_xc_interpreter.wasm");
+	let code_gateway = read_contract("cw_xc_gateway.wasm");
+	let out_dir =
+		option_env!("NIX_CARGO_OUT_DIR").unwrap_or(env!("OUT_DIR")).to_owned() + "/cw20_base.wasm";
```

Comment:
Alternatively:
```suggestion
		[option_env!("NIX_CARGO_OUT_DIR").unwrap_or(env!("OUT_DIR")), "/cw20_base.wasm"].concat();
```
It’s slightly longer but forgoes reallocation when concatenating.